scripts/azure_deployment/arm_aci.py

```diff
@@ -88,73 +89,132 @@ def make_aci_deployment(parser: ArgumentParser) -> Deployment:
+                        {
+                            "name": f"{args.deployment_name}-{i}",
+                            "properties": {
+                                "image": args.aci_image,
+                                "command": [
+                                    "/bin/sh",
+                                    "-c",
+                                    " && ".join(
+                                        [
+                                            *STARTUP_COMMANDS[args.aci_type](
+                                                args,
+                                                i,
                                             ),
-                                        ],
-                                        "ports": [
-                                            {"protocol": "TCP", "port": 8000},
-                                            {"protocol": "TCP", "port": 22},
-                                        ],
-                                        "environmentVariables": [],
-                                        "resources": {
-                                            "requests": {"memoryInGB": 16, "cpu": 4}
-                                        },
-                                    },
-                                }
-                            ],
-                            "initContainers": [],
-                            "restartPolicy": "Never",
-                            "ipAddress": {
+                                            "tail -f /dev/null",
+                                        ]
+                                    ),
+                                ],
                                 "ports": [
                                     {"protocol": "TCP", "port": 8000},
                                     {"protocol": "TCP", "port": 22},
                                 ],
-                                "type": "Public",
+                                "environmentVariables": [],
+                                "resources": {"requests": {"memoryInGB": 16, "cpu": 4}},
                             },
-                            "osType": "Linux",
-                        },
-                    }
-                    for i in range(args.count)
-                ],
-            },
+                        }
+                    ],
+                    "ipAddress": {
+                        "ports": [
+                            {"protocol": "TCP", "port": 8000},
+                            {"protocol": "TCP", "port": 22},
+                        ],
+                        "type": "Public",
+                    },
+                },
+            }
+            for i in range(args.count)
+        ],
+    }
+
+    if args.aci_deploy_external_executer:
+        # NOTE: It currently exposes the attestation-container's port (50051) to outside of the container group for e2e testing purpose,
+        # but it shouldn't be done in actual CCF application.
```

Comment:
Should the CLI argument be renamed accordingly, to indicate that it's for testing purposes only, e.g. `end-to-end-tests` or something?

Reply:
This should be resolved soon in https://github.com/microsoft/CCF/issues/4804, but it sounds good since it's currently used only for 2e2 testig